tests/requirements_tensorflow

```diff
@@ -0,0 +1,12 @@
+# tensorflow-intel inside tensorflow still requires numpy<2.0.0
+numpy==1.26.4
+pytest==7.0.1
+pytest-xdist[psutil]==3.6.1
+pytest-html==4.1.1
+transformers==4.45.1
+tensorflow==2.17.0
+# tensorflow-text is not available for both Windows and ARM platforms
+tensorflow-text==2.17.0; platform_system == "Linux" and platform_machine == "x86_64"
+tensorflow-hub==0.16.1
+jax==0.4.33
+defusedxml==0.7.1
```

Comment:
have you considered pyproject.toml files and using Poetry and python frontend / backend?